text/0002-keyboard-accessibility.md

```diff
@@ -0,0 +1,50 @@
+- Start Date: March 16, 2023
+- RFC PR: (leave this empty)
+- Issue: (leave this empty)
+
+# Summary
+
+We need a keyboard accessibility standard for Webstudio, since global standards only provide recommendations in many cases, but we need to be consistent.
+
+# Detailed design
+
+## Which elements should be focusable?
+
+All interactive elements should be focusable by keyboard. Interactive elements are those that do anything user-facing on mouse or keyboard interactions, like click or enter.
+
+Example: A button or a link should always be focusable, whereas a div that has no interactive usage not be focusable, unless it is a list item and has focusable elements inside.
+
+## Vertical lists of focusable items
+
+Long lists with focusable items are hard to exit if we tab through items. This is why once the first list element is focused, we choose to use arrow keys to navigate focus in the list.
+Tab will focus the next tabbable element after the list.
+
+## Nested focusable elements
+
+When you have an item that is focusable/interactive and then inside of that item you have more interactive elements, we have a choice to make.
+We can either let the user tab into the child elements or allow using arrow keys as a secondary mechanism to focus child elements.
+For consistency with lists, we choose to use arrow keys for all child elements and tab for focusing the next tabbable element that comes after the parent.
+
+## Vertical list with focusable items with nested focusable elements
+
+In this case, we have already decided to use vertical arrow keys to focus list items. To focus list item focusable child elements, we can now only use horizontal arrow keys.
+
+## Grids
```

Comment:
The pattern here IMO would be more akin to a [Grid Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/data-grids/).